Widgets/Widgets/PictureOfTheDayWidget.swift

```diff
@@ -159,7 +187,7 @@ struct PictureOfTheDayProvider: TimelineProvider {
 	}
 
 	func getSnapshot(in context: Context, completion: @escaping (PictureOfTheDayEntry) -> Void) {
-		dataStore.fetchLatestAvailablePictureEntry(usingImageCache: context.isPreview) { entry in
+		dataStore.fetchLatestAvailablePictureEntry(usingCache: context.isPreview) { entry in
```

Comment:
I have noticed Top Read was changed to `usingCache: true` here instead of considering `context.isPreview`. I'm not sure which way is best but I thought worth pointing out the discrepancy.

Reply:
I think we should probably change it in Top read to respecting `context.isPreview` – I'll make the change and spin up a PR and we can test it both ways. Of course the actual behavior is completely unknown to me, but I can imagine a scenario where the system maybe tries to opportunistically create a snapshot giving our widget a bunch of time to do whatever network operations we need, and if we aren't respecting that flag we may end up delivering less relevant information than we're actually able to. Apple's conceptual documentation on widgets does something similar for `getSnapshot` so probably worth sticking with the pattern until we discover it's wrong.